api/app/hfi/fire_centre_cache.py

```diff
@@ -25,38 +25,23 @@ async def get_cached_hydrated_fire_centres(station_codes: List[int]) -> Optional
     if cached_json:
         logger.info('redis cache hit %s', key)
         cache_string = json.loads(json.loads(cached_json))
```

Comment:
```suggestion
        cached_json = json.loads(cached_json)
```
Why the double load? Also - it's not a cache_string once json has loaded it - it's a dict.

Reply:
It's a string according to the runtime error I get with a single load:
```
    cached_response: HFIWeatherStationsResponse=HFIWeatherStationsResponse(**cache_string)
TypeError: app.schemas.hfi_calc.HFIWeatherStationsResponse() argument after ** must be a mapping, not str
```

Reply:
cache.get returns a binary array - you need:
```suggestion
        cache_string = json.loads(cached_json.decode())
```

Reply:
Tested the above solution with a clean redis cache and server, and I still get the same error

Reply:
So the first load decode it as string json, the second load decodes it as a python object. Not sure what's need to skip the first string load.

Reply:
Let's chat - I tried it and it worked... weird.